.github/release.yml

```diff
@@ -0,0 +1,21 @@
+changelog:
```

Comment:
If we want this to match our current headings we may need to make some slight changes here to make it match exactly. Although i am not sure it needs to match exactly.....what do you think @sedefsavas

Reply:
I agree, don't need to match.
I just added some nits. We could combine Failing tests and others categories.